api/openapi.yml

```diff
@@ -540,6 +540,46 @@ paths:
           description: Unauthorized
       security:
         - authelia_auth: []
+  /api/secondfactor/duo_devices:
+    get:
+      tags:
+        - Second Factor
+      summary: Second Factor Authentication - Duo Mobile Push
+      description: This endpoint retreives a users available devices and capabilities from Duo.
```

Comment:
Shouldn't this be something about retrieving the users preferred device/caps?

Reply:
This performs a PreAuth and retrieves available devices and capabilities, not the preferred device.

https://github.com/authelia/authelia/blob/5ad3b413fed657182efdc626bb955a1f1a5f6122/internal/handlers/handler_register_duo_device.go#L14